docs/docs/getting-started/install.md

```diff
@@ -119,24 +119,47 @@ The following [resource providers need to be registered](https://learn.microsoft
 
 By default, Constellation tries to register these automatically if they haven't been registered before.
 
-You need the following permissions for your user account:
-
-- `Contributor` (to create cloud resources)
-- `User Access Administrator` (to create a service account)
-
-If you don't have these permissions with scope *subscription*, ask your administrator to [create the service account and a resource group for your Constellation cluster](first-steps.md).
-Your user account needs the `Contributor` permission scoped to this resource group.
+To [create the IAM configuration](../workflows/config.md#creating-an-iam-configuration) for Constellation, you need the following permissions:
```

Comment:
```suggestion
To [automatically create the IAM configuration](../workflows/config.md#creating-an-iam-configuration) for Constellation, you need the following permissions:
```
To highlight that this refers to our feature so we don't confuse people who want to set these up manually.

Reply:
But doesn't manually creating it require the same permissions?

Reply:
True. Let’s just keep it as it is then 😄